optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/reflect/field/TestdataFieldAnnotatedSolution.java

```diff
@@ -75,6 +74,7 @@ public void setScore(SimpleScore score) {
     // Complex methods
     // ************************************************************************
 
+    @PlanningFactCollectionProperty
```

Comment:
We've just agreed on IRC that we're not changing this and sticking to method-based annotations, except for one example to demonstrate the capability.